README.md

```diff
@@ -86,7 +93,44 @@ If you're making a package within a community that has an existing package templ
 
 ## Notes for developers
 
+### Contributing
+
+The [website] and source pages are publicly available, but contributions are restricted to members and associate members of ARC.
+Bug reports (and fixes) are welcome from anyone!
+
+Pages all live in the `docs/pages` sub-directory, and are written in markdown.
+
+To contribute:
+
+1. Create a new branch.
+2. Modify an existing page, create a new one, or tweak the template.
+   a. Run [pre-commit](https://pre-commit.com/) which will lint your changes.
+   b. Check the tests pass if you modified the template (`pytest -s`).
+3. Open a pull request with changes.
+4. Someone who did not contribute to the PR should review it.
+5. If approved, the reviewer should merge the PR.
+6. If changes requested, the PR author should address the comments, then ask for review again.
+
+### Development
+
 - To run cookieninja using a specific branch of the template:
+
   ```
   cookieninja https://github.com/UCL-ARC/python-tooling --checkout <branch-name>
   ```
+
+- To run the tests locally:
+
+  ```
+  pytest -s
+  ```
+
+  We do a test cookiecut of a dummy package and install it to ensure the template setup works.
+
+- To build the docs locally
+
+  1.  [Install jekyll](https://jekyllrb.com/docs/installation/)
```

Comment:
```suggestion
  1.  [Install jekyll](https://jekyllrb.com/docs/installation)
```

Reply:
This isn't strictly true, right? You need to

1. Install `ruby`
2. Install `bundle`
3. Run `bundle install`
4. et voila

Reply:
I've never used `just-the-docs`. But for my own website these are the commands I would run